public/js/components/AtomEdit/CustomEditors/StoryQuestionFields/Question.js

```diff
@@ -94,7 +100,7 @@ export class StoryQuestionsQuestion extends React.Component {
         </ManagedForm>
```

Comment:
It adds clutter but would be good to give a prompt to the user that the answer has to be another atom
`<small> Answer the question with an atom, select one below </small>`

Reply:
That's a good idea, thanks 👍